site/ts/server.ts

```diff
@@ -4,7 +4,7 @@
  * @param {string[]} parts
```

Comment:
We don't need to mark this file as an entryPoint right for esbuild as we're not directly including this anywhere. It's always implicitly bundled by the other scripts, and esbuild handles that automatically for us.

As such, let's move this file under `site/ts/utils/` directory and only bundle files at the top level as "entryPoints". I can see us adding additional files like `server.ts` that are included across multiple entryPoints, but are not entryPoints themselves.